slsim/Pipelines/sl_hammocks_pipeline.py

```diff
@@ -82,6 +100,82 @@ def __init__(self, slhammocks_config=None, sky_area=None, cosmo=None):
             table = table_translator_for_slsim(table, cosmo)
 
             self._pipeline = table
+        skypy_config = os.path.join(module_path, "data/SkyPy/slhammock_skypy.yml")
+        with open(skypy_config, "r") as file:
+            content = file.read()
+        redshift_array = list(self._pipeline["z"].value)
+        old_z = "z: PLACEHOLDER_Z"
```

Comment:
the class does not explain in sufficient detail what is happening here and what the role of this class is. I guess there is a merge between skypy galaxies and SL_hammock happening?